app/assets/javascripts/sufia/save_work/save_work_control.es6

```diff
@@ -43,6 +43,17 @@ export class SaveWorkControl {
   }
 
   /**
+   * Keep the form from being submitted many times.
+   *
+   */
+  preventSubmitIfAlreadyInProgress() {
+    this.form.on('submit', (evt) => {
+      if (this.isValid())
```

Comment:
Is the validity check here just to prevent marking as disabled an already disabled control?